File: С/mario/Mario.c
#include <cs50.h>
#include <stdio.h>

int main(void) {
  int h;
  do {
    int h = get_int("Hight?\n");
  } while (h < 1 || h > 8);

  //a pyramid with the desired height
  for (int i = 0; i < h; i++) {
    //the number of spaces
    for (int j = h - 1 - i; j > 0; j--){
      printf(" ");
    }

    //the number of hashes
    for (int k = 0; k <= i; k++) {
      printf("#");
    }

    printf("\n");
  }
  return 0;
}

File: Python/HomeWork/mario/Mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

for floor in range(height):
    for space in range(height - floor - 1):
        print(" ", end = "")
    for hash in range(floor + 1):
        print("#", end = "")
    print()
